python/base/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM socrata/base
+
+MAINTAINER Nick Kypreos <nick.kypreos@socrata.com>
+
+RUN apt-get update
+
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y build-essential \
+    python-dev \
+    python-pip \
+    git \
+    libev4 \
+    libev-dev \
+    libpq-dev
+
+RUN pip install blist==1.3.6 \
+    cassandra-driver==2.1.4 \
```

Comment:
I've been out of the loop on the package inclusion question, but it seems that things like cassandra, elasticsearch, pep8, blist, and psycopg2 are all packages that we may not use.  I think there is a decent case to be made for the others living in base. 

However, the "right" answer is to build this into our own toolchain so we can pip and apt at build time...